plugins/org.eclipse.n4js.xpect.ui/src/org/eclipse/n4js/xpect/ui/common/XpectN4JSES5TranspilerHelper.java

```diff
@@ -125,7 +124,7 @@ public String doCompileAndExecute(final XtextResource resource,
 
 		loadXpectConfiguration(init, fileSetupContext);
 
-		File artificialRoot = Files.createTempDirectory("n4jsXpect").toFile();
+		Path artificialRoot = Files.createTempDirectory("n4jsXpectOutputTest");
```

Comment:
When will this temporary directory be cleaned up? Couldn't find it.

Reply:
I put this on my clean up list.